examples/example.c

```diff
@@ -93,9 +93,15 @@ main(int argc, char **argv)
             options, sentry_transport_new(print_envelope));
     }
 
+    if (has_arg(argc, argv, "capture-transaction")) {
+        sentry_options_set_traces_sample_rate(options, 1.0);
+    }
+
     sentry_init(options);
 
-    if (!has_arg(argc, argv, "no-setup")) {
+    if (!has_arg(argc, argv, "no-setup")
+        && !has_arg(argc, argv, "capture-transaction")) {
```

Comment:
why this change? We should still add all the scope to the transaction (and assert that via tests)

Reply:
was hoping somebody'd comment on this! looking back at this now, i think this can now be removed - in the past this actually used to break some of my testing while i was still working on getting transactions to work. the level setting, for example used to cause some issues for me. this isn't a problem anymore though, so this can be removed.